CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ## Unreleased
 
+### Breaking changes
+
+#### Check your favicons, app icons and OpenGraph image still work
+
+We've changed the names, formats and sizes of icon assets that we distribute in Frontend. You will want to check that the correct files are in place.
```

Comment:
```suggestion
We've changed the names, formats and sizes of icon assets that we distribute in GOV.UK Frontend. You will want to check that the correct files are in place.
```

Reply:
I made a couple of tweaks to Colin's suggestion